commands/web/check-annertech-ddev

```diff
@@ -11,8 +11,40 @@ YELLOW='\033[0;33m'
 NC='\033[0m' # No Color
 STARS='****************************************'
 
+# Get the current Addon version
 ADDONVER=$(grep -oP '(?<=version: ")[^"]+' .ddev/addon-metadata/annertech-ddev/manifest.yaml)
-LATESTVER=$(curl -sL https://api.github.com/repos/annertech/annertech-ddev/releases/latest | jq -r ".tag_name")
+
+# Define the path to store the version file
+VERSION_FILE="/tmp/addon-version.txt"
+
+# Function to get the latest tag from GitHub API using curl and jq
+get_latest_tag() {
+    # Use -s for silent mode, L for follow redirects, and output in JSON format
+    # Extract the tag_name field using jq and convert it to a raw string (without quotes)
+    curl -sL https://api.github.com/repos/annertech/annertech-ddev/releases/latest | \
+        jq -r ".tag_name"
+}
+
+# Check if version file exists
+if [ ! -f "$VERSION_FILE" ]; then
+    echo "Version file does not exist. Creating..."
+    # Get latest tag and save it to the version file
+    get_latest_tag > "$VERSION_FILE"
+else
+    # Calculate how long ago the version file was modified in seconds
+    age=$(( $(date +%s) - $(stat -c %Y "$VERSION_FILE") ))
+
+    if [ $age -gt 86400 ]; then
+        echo "Version file is older than 24 hours. Updating..."
+        # Get latest tag and save it to the version file
+        get_latest_tag > "$VERSION_FILE"
+    else
+        echo "Version file exists and is less than 24 hours old."
```

Comment:
Very verbose, does it have to be?
Useful for addon-developers, but I feel it is pointless to drupal developers

Reply:
I was in two minds whether to keep it. You only get one message added to the 25+ line stream so, on balance, thought it was OK to keep. It doesn't _need_ to be there though.

Reply:
It doesn't hurt, but I am not very fond of super-verbose things unless there is a need

Approved either way